grammars/fortran - free form.cson

```diff
@@ -2028,6 +2029,65 @@
         ]
       }
     ]
+  'procedure-definition':
+    'comment': 'Procedure program unit. Introduced in the Fortran 2008 standard.'
+    'name': 'meta.procedure.fortran'
+    'begin': '(?i)(?:^|(?<=;))(?=[^\'";!\\n]*\\bmodule\\s+procedure\\b)'
```

Comment:
Drop the `(?:^|(?<=;))` part here. It causes the rule to not work with the `Fortran - Fixed Form` rules. Also, since there are no attributes than can be placed in front of the `module procedure` statement you can do away with the whole outer begin/end pattern like in the subroutine-definition rules.